fog/ledger/test_infra/src/lib.rs

```diff
@@ -84,6 +87,58 @@ impl LedgerEnclave for MockEnclave {
     ) -> Result<(), mc_fog_ledger_enclave::Error> {
         unimplemented!()
     }
+
+    fn ledger_store_init(&self, _ledger_store_id: ResponderId) -> EnclaveResult<NonceAuthRequest> {
```

Comment:
When do these methods get implemented?

Reply:
I was following the structure already present in test_infra - all of the methods in `impl LedgerEnclave for MockEnclave` were just unimplemented when Andrew and I started this fork. It looks like this is still the case in `master`. It's being used as a stub for a Merkele proof test in `merkle_proof_service.rs`. So, from what I can tell, all of these methods being unimplemented is intended - it's there to get the type system to play nice with certain tests.